content/en/tracing/setup_overview/open_standards/_index.md

```diff
@@ -93,6 +93,371 @@ service:
       exporters: [datadog/api]
 ```
 
+### Environment Specific Setup
+
+#### Host:
+
+- Download the appropriate binary from [the project repository latest release](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/latest).
+
+- Create a `otel_collector_config.yaml` file. Here is an example template to get started. It enables the collector's otlp receiver and datadog exporter.
+
+- Run on the host with the configration yaml file set via the `--config` parameter. For example,
+
+  ```
+    curl -L https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/latest/download/otelcontribcol_linux_amd64 | otelcontribcol_linux_amd64 --config otel_collector_config.yaml
```

Comment:
If they downloaded the binary, they wouldn't use `curl` right?

Reply:
yup good catch. this was combining downloading the release and running it...will remove the `curl` bit to make it more clear.